falcosidekick/CHANGELOG.md

```diff
@@ -5,6 +5,28 @@ numbering uses [semantic versioning](http://semver.org).
 
 Before release 0.1.20, the helm chart can be found in `falcosidekick` [repository](https://github.com/falcosecurity/falcosidekick/tree/master/deploy/helm/falcosidekick).
 
+## 0.6.3
```

Comment:
This PR is missing https://github.com/falcosecurity/falcosidekick/pull/421/files#